### ISSUE TITLE
unable to read user table in voiceover

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#1503]
*Issue migrated from trac ticket [#1282](http://colloquy.info/project/ticket/1282)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** High | **version:** 2.1 (Mac) | **status:** closed | **resolution:** Duplicate | **keywords:** voiceover, accessibility, blind users
___
#### 2008-09-30 02:48:24: Shawn created the issue

Voiceover reads all rows in the table of users in the drawer which is reading as blank. 
If it weren't for this, I would use coloquy exclusively instead of limechat. 

___
#### 2008-09-30 02:50:11: Shawn commented

This table does not read at all. 

___
#### 2008-09-30 06:15:31: @janeylicious commented

related to #998 just for future reference

___
#### 2008-09-30 18:02:57: Shawn commented

Either because of the update to the client or voiceover, the client reads ok just about everywhere else except for this table. it appears great strides  have taken place with voiceover  and coloquy  in the past two years. Certainly coloquy is much more usable than the description give of ticket 998. 

___
#### 2008-10-01 18:54:00: Shawn commented

never the less i would still like to see this issue addressed. 

___
#### 2008-10-01 18:54:00: Shawn changed priority from *normal* to *high*

___
#### 2009-04-16 23:45:08: @xeenon changed status from *new* to *closed*

___
#### 2009-04-16 23:45:08: @xeenon changed resolution from ** to *duplicate*

___
#### 2009-04-16 23:45:08: @xeenon commented

Considering as a dupe of #998.
